metricflow-semantics/metricflow_semantics/model/semantic_manifest_lookup.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, semantic_manifest: SemanticManifest) -> None:
         self._time_spine_sources = TimeSpineSource.build_standard_time_spine_sources(semantic_manifest)
         self.custom_granularities = TimeSpineSource.build_custom_granularities(list(self._time_spine_sources.values()))
         self._semantic_model_lookup = SemanticModelLookup(
-            model=semantic_manifest, custom_granularities=self.custom_granularities
+            semantic_manifest=semantic_manifest, custom_granularities=self.custom_granularities
```

Comment:
this has been bugging me for so long and I never got around to fixing it haha thank you!